IO-pc-local/angular.json

```diff
@@ -30,10 +30,8 @@
             ],
             "scripts": [],
             "server": "src/main.server.ts",
-            "prerender": true,
-            "ssr": {
-              "entry": "server.ts"
-            }
+            "prerender": false,
+            "ssr": false
```

Comment:
Czy po usunięcie SSR zniknęły problemy, które miałeś?

Reply:
Tak. Jakbyś w tym miejscu przywrócił oryginalną treść, to w konsoli angulara sypie błędami (mimo że po stronie klienta chyba działa). Jakby Ci się chciało to możesz przetestować, może będziesz wiedział co z tym zrobić